packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -178,12 +178,12 @@ def test_octarine_srv(cluster, timeout=30):
 def test_pkgpanda_api(cluster):
 
     def get_and_validate_package_ids(node, path):
```

Comment:
Would you prefer refactoring this `get_and_valid_package_ids` signature to read `path, node` because that's how it is being invoked further down and will help greatly in readability.